src/main/resources/assets/twilightforest/lang/zh_cn.lang

```diff
@@ -277,7 +283,7 @@ item.steeleafShovel.name=钢叶铲
 item.steeleafPick.name=钢叶镐
 item.steeleafAxe.name=钢叶斧
 item.steeleafHoe.name=钢叶锄
-item.minotaurAxe.name=米诺陶诺斯战斧
+item.minotaurAxe.name=米诺陶战斧
 item.mazebreakerPick.name=迷宫破坏者
 item.transformPowder.name=转换粉
 item.meefRaw.name=生牛头人肉
```

Comment:
Do you need to fix this, too?

Reply:
Isn't meef == minotaur + beef?

Reply:
Yes. Alternatively it could also be man+beef. Up to you!

Reply:
I thought it'd be minoshroom's meat.

Reply:
Seems like you @liach forget the fact that minoshroom drops stroganoff (also the minotaur axe), which... corresponding to the mushroom stew.

Reply:
eh, I haven't played tf a lot, in fact.